src/shared/behaviors/nonscrollable.html

```diff
@@ -7,6 +7,35 @@
 
 */
 (function(scope) {
+
+	var _openedInstances = [];
```

Comment:
Maybe use a `new Map()` (or a `new Set()`), because the data structure is more relevant